### ISSUE TITLE
Publish an official Docker Image in ghcr.

### ISSUE THREAD

**Issue by luisalrp** [elizaOS/eliza#876]
Would be really useful to have an official Docker Image to pull.

**Describe the solution you'd like**

A Github Action that creates the Docker Image with each release, tag it properly and upload it to ghcr or dockerhub.

**Describe alternatives you've considered**

An alternative is really ugly. It is being aware of each release and do it manually or use some cicd to do it.

Thanks!

**Comment by v1xingyue** [comment]
#889

**Comment by luisalrp** [comment]
Hey @v1xingyue,

Thanks for the PR. I noticed there was a release, but I can't see the package, while if I go to the [Create and publish a Docker image action](https://github.com/ai16z/eliza/actions/workflows/image.yaml), I can see they were ok. Not sure whats happening. Maybe its a rights problem?
<img width="1287" alt="image" src="https://github.com/user-attachments/assets/5debd6ab-a124-4063-8442-12d90e41efa8" />

Thanks!

**Comment by v1xingyue** [comment]
Ya @luisalrp I checked the log action running. It tells image has been pushed:

![image](https://github.com/user-attachments/assets/990ea090-9810-40ca-859a-03262fe4c729)

But When I pull one image from my cloud server. 
It tells no unauthorized :

![image](https://github.com/user-attachments/assets/cea26f06-054b-4ea3-ad9c-bc5e4da27063)

I'm not sure, is it the problem of permission?

**Comment by v1xingyue** [comment]
![image](https://github.com/user-attachments/assets/11b26320-f94c-46f5-80a0-32ad4460e3e5)

Also github  shows we have one public image. But we can't see. Maybe  we should check the settings about package.

**Comment by luisalrp** [comment]
Yeah, I think its something about visibility (sorry for using rights instead of visibility). When I tried to pull it, I got the same unauthorized as you. Not sure how to solve it.

I went to [Solve Problems in Work with containers registry](https://docs.github.com/es/packages/working-with-a-github-packages-registry/working-with-the-container-registry#troubleshooting), and there is nothing about it.

I [asked xAi](https://x.com/i/grok/share/dPgqYWjQgpJ8IYIo8MEIO5ojM) and it says that packages by default sets to private.

I hope it helps. Thanks!

**Comment by luisalrp** [comment]
Hey, @v1xingyue, @lalalune;

I created a PR to change package visibility when creating new releases. I'm happy to have contributed a little bit.

Best, Luis.

**Comment by v1xingyue** [comment]
Wow thanks that's so cool

**Comment by v1xingyue** [comment]
> Hey, @v1xingyue, @lalalune;
> 
> 
> 
> I created a PR to change package visibility when creating new releases, so no need to manually change visibility on future releases. I'm happy to have contributed a little bit.
> 
> 
> 
> Best, Luis.

Hey, your pr about change visibility runs failed , it returns 401

**Comment by luisalrp** [comment]
Sorry for that. Then it can be done only by config. [Configuring a package's access control and visibility](https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility).

I created a PR to delete the non working task.

**Comment by v1xingyue** [comment]
Ya that's really true. So let's change the package visibility @lalalune

**Comment by luisalrp** [comment]
Hey frens!

Any ETA for this? It is a pity that being done, it can't be used due to package visibility config. @v1xingyue, do you know if there is someone else with the rights to change package visibility? I understand that with Christmas and road show in Asia it is being busy days for @lalalune 😊

Best, Luis!

**Comment by luisalrp** [comment]
Maybe @shakkernerd can change package visibility? I've seen publishing the releases 😃

**Comment by Kevin-Mok** [comment]
Also hoping to try using the Docker package soon!

**Comment by shakkernerd** [comment]
> Maybe [@shakkernerd](https://github.com/shakkernerd) can change package visibility? I've seen publishing the releases 😃

I will take a look at this.

**Comment by Kevin-Mok** [comment]
Was there a reason this didn't pan out?
